{{cookiecutter.project_name}}/environments/kolla/configuration.yml

```diff
@@ -30,9 +30,9 @@ openstack_service_workers: 5
 es_heap_size: "4g"
 
 # neutron
-neutron_plugin_agent: "openvswitch"
-
-enable_neutron_agent_ha: "yes"
+neutron_plugin_agent: "ovn"
+neutron_ovn_availability_zones:
```

Comment:
Can you do this like follows:
```
neutron_ovn_availability_zones: [nova]
```
This makes it much easier to change it later by just replacing this line